### ISSUE TITLE
oa_ngrams is not working due to openalex api

### ISSUE THREAD

**Issue by adrientaudiere** [ropensci/openalexR#281]
I rerun some script that worked a few weeks ago, and it seems that oa_ngrams is broken. I tried the example from the help page and it returns nothing. Maybe the ngrams api of openalex is no longer available as discuss in their [documentation](https://docs.openalex.org/api-entities/works/get-n-grams#api-endpoint).

```
ngrams_data <- oa_ngrams(
    works_identifier = c("W1964141474", "W1963991285"),
    verbose = TRUE
)
```

**Comment by yjunechoe** [comment]
Thanks - we're aware of this problem and watching it closely! Since they say "The n-gram API endpoint is not _currently_ in service", that seems to suggest it'll come back online - we can leave this issue open to track this.

BTW @trangdata do you know if OpenAlex have a status dashboard of sorts for their services that we can query/scrape?

**Comment by yjunechoe** [comment]
Yeah - something like GitHub status! https://www.githubstatus.com/

**Comment by adrientaudiere** [comment]
Ok, thank you kindly for your reactivity and this R package.
